### ISSUE TITLE
Scrollable view should still scroll while sliding is disabled 

### ISSUE THREAD

**Issue by ikresoft** [umano/AndroidSlidingUpPanel#643]
How to disable sliding?

**Comment by tokudu** [comment]
setSlidingEnabled(false)

**Comment by ikresoft** [comment]
Sorry, but this method is unknown?

**Comment by iampawan** [comment]
@tokudu there is no such method?

**Comment by tokudu** [comment]
Use setEnabled

**Comment by ikresoft** [comment]
If I use setEnabled(false) and have ListView inside then ListView not reacting.

**Comment by tokudu** [comment]
How about `setTouchEnabled`?

**Comment by Doctoror** [comment]
I came across similar problem. Content of the layout not always reacting to touch events if touching is disabled. The fix is not intercepting touch event if !isTouchEnabled()

```java
    @Override
    public boolean onInterceptTouchEvent(MotionEvent ev) {
        // If the scrollable view is handling touch, never intercept
        if (mIsScrollableViewHandlingTouch || !isTouchEnabled()) { //This line, move to return false if touch not enabled
            mDragHelper.cancel();
            return false;
        }
        ...
```

**Comment by tokudu** [comment]
Fixed in https://github.com/umano/AndroidSlidingUpPanel/commit/966fee4f049c43c1fded6371d1501d761f043395. Will be released in 3.3
